src/test/electron-config.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
```

Comment:
it seems there is a lot of duplicated code between electron and browser, as well as electron-main and node. It would be great to dry those up